src/curriculum/level1/learningLetters.js

```diff
@@ -264,6 +264,53 @@ const lesson7 = {
   ],
 };
 
+const lesson8 = {
+  letters: {
+    ⲥ: getLetterExplanation('ⲥ'),
+    ⲧ: getLetterExplanation('ⲧ'),
+    Ϫ: getLetterExplanation('Ϫ'),
+  },
+  exampleWords: [
+    {word: 'Ϩⲓϫⲉⲛ', pronunciation: 'aghapee'},
+    {word: 'Ⲡⲗⲁⲥⲙⲁ', pronunciation: 'psyshee'},
+    {word: 'Ⲉⲛⲧⲟⲗⲏ', pronunciation: 'qeemi'},
+    {word: 'Ⲥⲙⲟⲩ', pronunciation: 'kyryos'},
+    {word: 'Ϫⲱⲣ', pronunciation: 'sheroobim'},
+    {word: 'Ⲉⲛⲧⲏⲙⲁ', pronunciation: 'sheroobim'},
+    {word: 'Ⲕⲟⲥⲙⲟⲥ', pronunciation: 'sheroobim'},
+    {word: 'Ⲥⲙⲏ', pronunciation: 'sheroobim'},
```

Comment:
Pronunciation is incorrect

Reply:
Should there be a jikim on this word? i.e. Ⲥ̀ⲙⲏ

Reply:
same as above